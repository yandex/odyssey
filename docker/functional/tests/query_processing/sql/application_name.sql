SELECT 'z';

SET application_name to 'AAA';
SHOW application_name;

SET application_name to 'This-is-long-application-name-that-must-be-cut-to-add-host-and-this-is-tail';
SHOW application_name;