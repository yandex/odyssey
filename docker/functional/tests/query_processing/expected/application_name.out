SELECT 'z';
 ?column? 
----------
 z
(1 row)

SET application_name to 'AAA';
SHOW application_name;
 application_name 
------------------
 AAA - 127.0.0.1
(1 row)

SET application_name to 'This-is-long-application-name-that-must-be-cut-to-add-host-and-this-is-tail';
SHOW application_name;
                        application_name                         
-----------------------------------------------------------------
 This-is-long-application-name-that-must-be-cut-to-a - 127.0.0.1
(1 row)

